src/vnc.yml

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
Can you please add a comment in here (or in `src/playbook.yml`, wherever you feel it's more appropriate) explaining that we are installing the VNC server in order to add the VNC ssh user and not to actually use the VNC server itself?

Reply:
Please see commit 00cd8b1.

Reply:
Thanks!